src/features/catalog/CatalogPage.tsx

```diff
@@ -1,9 +1,20 @@
+import Box from '@mui/material/Box';
 import ProductList from './components/ProductList/ProductList';
 
 export default function Catalog() {
   return (
-    <>
+    <Box
+      component="div"
+      sx={{
+        maxWidth: { lg: '1108px', md: '850px', xs: '90%' },
+        display: 'flex',
+        flexDirection: 'column',
+        gap: 2,
+        margin: 'auto',
+        py: { lg: 8, md: 6, xs: 4 },
+      }}
+    >
       <ProductList />
```

Comment:
For future tasks this code will help us:
```suggestion
     <Grid container spacing={3}>
        <Grid size={3}>
          <div>Filters…</div>
        </Grid>
        <Grid size={9}>
          <ProductList />
        </Grid>
      </Grid>

Reply:
Optional/